{
  "uid" : "67a41f78bbb2de6d",
  "name" : "test_two",
  "fullName" : "test_py_02.TestClass#test_two",
  "historyId" : "6c2bcb20a9b0e140b3abc73842ff2e26",
  "time" : {
    "start" : 1639446796370,
    "stop" : 1639446796370,
    "duration" : 0
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL\n\nAssertionError: assert False\n +  where False = hasattr('hello', 'check')",
  "statusTrace" : "self = <py_test.test_py_02.TestClass object at 0x00000000038AF0F0>\n\n    @pytest.mark.xfail()\n    def test_two(self):\n        x = \"hello\"\n>       assert hasattr(x, 'check')\nE       AssertionError: assert False\nE        +  where False = hasattr('hello', 'check')\n\ntest_py_02.py:28: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 1,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "class_scope",
    "time" : {
      "start" : 1639446796369,
      "stop" : 1639446796370,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "skipped",
    "statusMessage" : "XFAIL\n\nAssertionError: assert False\n +  where False = hasattr('hello', 'check')",
    "statusTrace" : "self = <py_test.test_py_02.TestClass object at 0x00000000038AF0F0>\n\n    @pytest.mark.xfail()\n    def test_two(self):\n        x = \"hello\"\n>       assert hasattr(x, 'check')\nE       AssertionError: assert False\nE        +  where False = hasattr('hello', 'check')\n\ntest_py_02.py:28: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "985049b5129708fd",
      "name" : "stdout",
      "source" : "985049b5129708fd.txt",
      "type" : "text/plain",
      "size" : 12
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.xfail"
  }, {
    "name" : "suite",
    "value" : "test_py_02"
  }, {
    "name" : "subSuite",
    "value" : "TestClass"
  }, {
    "name" : "host",
    "value" : "FXDN-20170826XB"
  }, {
    "name" : "thread",
    "value" : "9588-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_py_02"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "323b978fa12db1fc",
      "status" : "skipped",
      "statusDetails" : "XFAIL\n\nAssertionError: assert False\n +  where False = hasattr('hello', 'check')",
      "time" : {
        "start" : 1639445438993,
        "stop" : 1639445438993,
        "duration" : 0
      }
    } ],
    "categories" : [ ],
    "tags" : [ "@pytest.mark.xfail" ]
  },
  "source" : "67a41f78bbb2de6d.json",
  "parameterValues" : [ ]
}